lookout/style/format/rules.py

```diff
@@ -272,19 +281,26 @@ def _prune_branches(cls, rules: Sequence[Rule],
         # TODO(vmarkovtsev): implement this function
         return rules
 
-    @classmethod
-    def _build_instances_index(cls, rules: Sequence[Rule],
-                               X: numpy.ndarray) -> Dict[int, Set[int]]:
+    def _build_instances_index(self, rules: Sequence[Rule], X: numpy.ndarray,
+                               leaf2rule: Sequence[Mapping[int, int]]) -> Dict[int, Set[int]]:
+
+        self.log.debug('building instances index')
```

Comment:
Oops sorry missed the single quotes here

https://github.com/src-d/guide/blob/master/engineering/conventions-python.md#code-style